### PR TITLE
de-UKify service and template

### PR DESCRIPTION
**Pull request by jimmoffet**

### CODE REVIEW

File: app/config.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ class Development(Config):
 
     NOTIFY_ENVIRONMENT = 'development'
     NOTIFY_LOG_PATH = 'application.log'
-    NOTIFY_EMAIL_DOMAIN = "dispostable.com"
+    NOTIFY_EMAIL_DOMAIN = "notify.sandbox.10x.gsa.gov"
```
Comment (rahearn) [inline app/config.py:437]: Does this need to be updated elsewhere to be accurate for staging?

File: app/config.py [hunk 2]
```diff
@@ -434,7 +434,7 @@ class Development(Config):
 
     NOTIFY_ENVIRONMENT = 'development'
     NOTIFY_LOG_PATH = 'application.log'
-    NOTIFY_EMAIL_DOMAIN = "dispostable.com"
+    NOTIFY_EMAIL_DOMAIN = os.getenv('NOTIFY_EMAIL_DOMAIN', 'notify.sandbox.10x.gsa.gov')
```
Comment (jimmoffet) [inline app/config.py:437]: this is the change

File: app/config.py [hunk 3]
```diff
@@ -533,7 +533,6 @@ class Staging(Config):
 
 
 class Live(Config):
-    NOTIFY_EMAIL_DOMAIN = os.environ.get('NOTIFY_EMAIL_DOMAIN')
```
Comment (rahearn) [inline app/config.py:536]: Where is this being set now? My understanding (which could be wrong, I'm a python noob) is that the prod version will basically set everything in the `Config` class, and then override anything in `Live`. Now neither of those areas have a setting for `NOTIFY_EMAIL_DOMAIN`

Comment (jimmoffet) [reply app/config.py:536]: Shows you what I get for pushing code during meetings. Meant to directly set notify.sandbox.10x... in `Config` and set the env with a default in `Development`. Should be fixed now.

**Review by rahearn — APPROVED** [review]
🎉

**Pull request merged**